catalog/app/containers/Bucket/UploadDialog.js

```diff
@@ -219,11 +221,12 @@ function FilesInput({ input, meta, uploads, setUploads, errors = {} }) {
     setUploads(R.dissoc(path))
   }
 
+  const onInputChange = input.onChange
```

Comment:
`input.onChange` used in onDrop callback too. Maybe you want to use `onInputChange` there too

Reply:
k